pkg/web/templates/layouts/base.html

```diff
@@ -9,6 +9,7 @@
 
   <main>
     {{ block "content" . }}{{ end }}
+    <div id="toast" class="toast toast-top toast-end"></div>
```

Comment:
`min-width: fit-content` is the default style applied to the toast elements. Should we update the error responses for the add counterparty modal? 

If not, we may have to amend the styling since attempting to override it with `!important` didn't work.

Reply:
I wasn't having much luck positioning the toasts either. In the counterparty modal, where does the toast appear, in the modal or outside it under main?